packages/upload/test/file.common.js

```diff
@@ -113,4 +134,33 @@ describe('<vaadin-upload-file> element', () => {
       expect(fileElement.hasAttribute('focus-ring')).to.be.false;
     });
   });
+
+  describe('disabled', () => {
```

Comment:
Button disabled states could be covered by snapshots, but those only test the Polymer version, and the binding for the disabled state is duplicated between Polymer and Lit versions.